src/ext/oblivc/copy.c

```diff
@@ -193,18 +172,6 @@ void ocShareMuxes(ProtocolDesc* pd,char* z,
   { ctx->recver.recv(ctx->recver.recver,tr,c,n,eltsize);
     ctx->sender.send(ctx->sender.sender,t,t2,n,eltsize);
   }
-#else
-  struct CorrFunXorArgs a = {.len=eltsize,.mask=t2};
-  if(protoCurrentParty(pd)==1)
-  { genrGateToShare(pd,ctx,t,t2,wc,n,eltsize);
-    //honestCorrelatedOTExtSend1Of2(ctx->sender,t,t3,n,eltsize,corrFunXor,&a);
-    honestCorrelatedOTExtRecv1Of2(ctx->recver,tr,c,n,eltsize);
-  }else
-  { evalGateToShare(pd,ctx,tr,wc,n,eltsize);
-    //honestCorrelatedOTExtRecv1Of2(ctx->recver,tr,c,n,eltsize);
-    honestCorrelatedOTExtSend1Of2(ctx->sender,t,t3,n,eltsize,corrFunXor,&a);
```

Comment:
Yes, this is in response to your previous [comment](https://github.com/samee/obliv-c/pull/22#discussion_r101928250). Perhaps I misinterpreted? The main obliv-c code uses the OTSender and OTRecver constructs, so I kept the branch that was compatible with those. To be honest, I'm not sure what the implications are. Perhaps I should have done something else. If so, advise.